ui/js/controllers/perf/alerts.js

```diff
@@ -506,6 +507,31 @@ perf.controller('AlertsCtrl', [
                     $scope.alertSummaries = _.union($scope.alertSummaries,
                                                     alertSummaries);
                 }
+
+                // Filling in downstream alertSummaries
+                _.forEach($scope.alertSummaries, function(alertSummary) {
```

Comment:
You should put this log inside the _.forEach logic above. There shouldn't be a need to iterate over $scope.alertSummaries twice.